lib/Widgets/DynamicNotebook.vala

```diff
@@ -668,6 +695,45 @@ namespace Granite.Widgets {
         // Use temporary field to avoid breaking API this can be dropped while preparing for 0.4
         string _add_button_tooltip;
 
+        public string add_button_tooltip_markup {
+            owned get { return add_button.get_tooltip_markup (); }
+            set { add_button.tooltip_markup = value; }
+        }
+
+        /**
+         * The text shown in the close button tooltip
+         */
+        private string _close_button_tooltip = _("Close Tab");
```

Comment:
Is this necessary for having the accel array? I feel like this might be out of scope and suited for another branch/discussion

Reply:
Could you clarify the controversy? Do you think the default text should be empty for now?
The text pre-dates this PR:
https://github.com/elementary/granite/pull/236/files?utf8=%E2%9C%93&diff=unified#diff-11a3fe7bae677d4582a9515547517dc9L238

Reply:
Yeah I'm just wondering what's the use case behind allowing developers to change this tooltip text? Shoudn't it always be "Close Tab"?

Reply:
Ah fair point. I was thinking it would allow people to use something more specific than "tab", like "Close file", "Close "image" etc. but I'll admit that's a bit of a stretch... 
We could easily remove the text properties (also for the add button).

Reply:
Yeah, I think we should keep the diff here small and focused on just adding the shortcut accels